src/libs/analytics.ts

```diff
@@ -46,9 +46,13 @@ const logAllModulesSeen = (numberOfModules: number) =>
 const logAllTilesSeen = (moduleName: string, numberOfTiles: number) =>
   firebaseAnalytics.logEvent(AnalyticsEvent.ALL_TILES_SEEN, { moduleName, numberOfTiles })
 
-const logConsultOffer = (offerId: number, moduleName: string) =>
-  firebaseAnalytics.logEvent(AnalyticsEvent.CONSULT_OFFER, { offerId, moduleName })
-
+const logConsultOffer = (offerId: number, moduleName: string | null, query?: string | '') => {
```

Comment:
```suggestion
const logConsultOffer = (params: {offerId: number, moduleName?: string, query?: string}) => {
  firebaseAnalytics.logEvent(AnalyticsEvent.CONSULT_OFFER, params)
}
```

ça donne quoi niveau Firebase? comment ça gère les undefined?

Reply:
```js
const logConsultOffer = (params: { offerId: number; moduleName?: string; query?: string }) =>
  firebaseAnalytics.logEvent(AnalyticsEvent.CONSULT_OFFER, params)
```